using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
+. **Acquire Red Hat Subscriptions** - Ensure that you have the relevant subscriptions for OpenShift as well as a subscription for xPaaS Middleware.
 
-* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead.
-// This is based on https://issues.jboss.org/browse/CLOUD-56
+. **Install OpenShift** - Before using the xPaaS JBoss EAP images, you must have an OpenShift environment installed and configured:
 
-ifdef::openshift-enterprise[]
-== Using the JBoss EAP xPaaS Image Streams and Application Templates
+.. The link:../install_config/install/quick_install.html method allows you to install OpenShift using an interactive CLI utility.
 
-The Red Hat xPaaS middleware images were
-link:../../install_config/imagestreams_templates.html[automatically created during the installation]
-of OpenShift along with the other default image streams and templates.
-endif::[]
+.. The link:../install_config/install/advanced_install.html method allows you to install OpenShift using a reference configuration. This method is best suited for production environments.
 
-[[Making-Configuration-Changes-EAP]]
-== Running and Configuring the JBoss EAP xPaaS Image
+.. Install and Deploy Docker Registry - Install the Docker Registry and then ensure that the Docker Registry is deployed to locally manage images as follows:
++
+----
+$ oadm registry --config=/etc/origin/master/admin.kubeconfig --credentials=/etc/origin/master/openshift-registry.kubeconfig
+----
++
+For further information, see link:../install_config/install/docker_registry.html
 
-You can make changes to the JBoss EAP configuration in the xPaaS image using either the S2I templates, or by using a modified JBoss EAP xPaaS image.
+. *Deploy a Router* - Use the instructions at the link:../install_config/install/deploy_router.html page for this step.
 
-=== Using the JBoss EAP xPaaS Image Source-to-Image (S2I) Process
+. *Privileges* - Ensure that you can run the oc create command with link:../architecture/additional_concepts/authorization.html#roles[cluster-admin] privileges
 
-The recommended method to run and configure the OpenShift JBoss EAP xPaaS image is to use the OpenShift S2I process together with the application template parameters and environment variables.
+. *Create Image Streams* - Create the xPaaS Middleware set of image streams for both versions of the xPaaS JBoss EAP image as follows:
++
+----
+$ oc create -f $XPAASSTREAMDIR/jboss-image-streams.json -n openshift
+----
++
+[NOTE]
+====
+For further information about creating image streams, see link:../install_config/imagestreams_templates.html
+====
 
-The S2I process for the JBoss EAP xPaaS image works as follows:
+. *Create InstantApp Templates* - InstantApp templates define a full set of objects for running applications.
```

Comment:
Again, If OpenShift was installed via quick or advanced installation, this step will have been automatically done already. You may want to reword this.